app/frontend/src/components/Navigation/Navigation.tsx

```diff
@@ -72,36 +63,82 @@ const useStyles = makeStyles((theme) => ({
       paddingRight: theme.spacing(3),
       width: "auto",
     },
+    display: "flex",
     flex: 1,
     justifyContent: "space-evenly",
     padding: 0,
   },
+  drawerPaper: {
+    padding: theme.spacing(2),
+    width: drawerWidth,
+  },
+  drawerHeader: {
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  drawerTitle: {
+    alignSelf: "center",
+    fontWeight: "bold",
+    paddingLeft: theme.spacing(1),
+  },
   gutters: {
     [theme.breakpoints.up("md")]: {
       paddingLeft: theme.spacing(3),
       paddingRight: theme.spacing(3),
     },
+    justifyContent: "space-between",
     paddingLeft: 0,
     paddingRight: 0,
   },
-  search: {
-    [theme.breakpoints.down("sm")]: {
-      marginInlineEnd: theme.spacing(1),
-    },
+  nav: {
+    display: "flex",
+  },
+  icon: {
+    marginLeft: theme.spacing(1),
+  },
+  bug: {
     alignItems: "center",
     display: "flex",
-    flexGrow: 3,
     justifyContent: "flex-end",
+    [theme.breakpoints.down("md")]: {
+      paddingRight: theme.spacing(2),
+    },
   },
   title: {
+    alignSelf: "center",
     fontWeight: "bold",
   },
 }));
 
 export default function Navigation() {
   const classes = useStyles();
-
   const authenticated = useAuthContext().authState.authenticated;
+  const [open, setOpen] = React.useState(false);
+
+  const logoutButton = <NavButton route={logoutRoute} label={LOG_OUT} />;
+
+  const drawerItems = (
+    <div>
+      <List>
+        {menu.map(({ name, route }) => (
+          <ListItem button key={name}>
+            <NavButton route={route} label={name} labelVariant="h2" />
```

Comment:
I think the drawer needs to close on click. Maybe a `closeDrawer` prop for `NavButton`?